### ISSUE TITLE
Discord Server

### ISSUE THREAD

**Issue by WarutaShinken** [MOONCOINTEAM/moon-blockchain#1]
Hi, since most of the fork community resides on Discord, could you make a Discord server?

**Comment by Yamanipanuchi** [comment]
Agreed! Communication is very useful and discord is a great location!

**Comment by SirCotare** [comment]
yep, you really need Discord
